devicetypes/smartthings/zigbee-accessory-dimmer.src/zigbee-accessory-dimmer.groovy

```diff
@@ -23,6 +23,7 @@ metadata {
 		capability "Configuration"
 
 		fingerprint profileId: "0104", inClusters: "0000,1000,0003", outClusters: "0003,0004,0005,0006,0008,1000,0019", manufacturer: "Aurora", model: "Remote50AU", deviceJoinName: "Aurora Wireless Wall Remote"
+		fingerprint manufacturer: "IKEA of Sweden", model: "TRADFRI wireless dimmer", deviceJoinName: "IKEA TRÅDFRI Wireless dimmer" // 01 [0104 or C05E] 0810 02 06 0000 0001 0003 0009 0B05 1000 06 0003 0004 0006 0008 0019 1000
```

Comment:
First one I've seen in the wild using this format. I think I like this format. What do you guys think? Still OK with this?
@dkirker @greens @varzac .

Reply:
I kinda want to clean the comment up a little bit. Something more like:

`// 01 [0104 or C05E] 0810 02 in: 0000 0001 0003 0009 0B05 1000 out: 0003 0004 0006 0008 0019 1000`

Since I'm not a computer I don't really care too much about the length of each array of clusters; replacing the lengths with keys makes it a little more readable (otherwise, the hex values seem to blur together).

But having the data can be useful.